app/controllers/comments/votes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Comments::VotesController < ApplicationController
+  before_action :ensure_current_user
+  before_action :set_comment
+
+  def create
+    current_user.likes @comment
+    if @comment.parent_id
+      closest_comment_by_votes = @comment.siblings.min_by {|c| (c.cached_votes_up - @comment.cached_votes_up).abs }
+      if @comment.cached_votes_up >= closest_comment_by_votes.cached_votes_up
```

Comment:
Won't this make it so that it will be ordered by vote, and then newest first, since you're prepending the new comment to comments with similar vote scores?